### ISSUE TITLE
Change default key to #aria.widgets.CfgBeans from /#aria.core.JsObject 

### ISSUE THREAD

**Issue by jakub-g** [ariatemplates/apidoc#14]
It seems people who open apidocs more often look for widgets documentation than JSObject methods. At least this is the case for myself, @carlo-mr and @lsimone :)

**Comment by ymeine** [comment]
I would set the initial page to what makes more sense, not what's more usual for some users. Except if of course we have a great majority of users tending to the same page preference **and if** we consider that setting the initial page to it actually makes sense.

Otherwise we should either let the user use a solution on its side, or provide a customization functionality (even a simple one with just one configuration option).

To me, having a home page would make more sense ^^ I don't know, like: 
- [DevDocs](http://devdocs.io/)
- [API Documentation - Ext JS - Sencha Docs](http://docs.sencha.com/extjs/5.0.1/)
- [Backbone.js](http://backbonejs.org/)
- etc.

**Comment by jakub-g** [comment]
Yeah it could also be some homepage with a few most interesting APIDOC links and some overview.

We could for instance implement some tracking on AT.com and figure out the list of most often visited hashes ;)

Also some links from our README in https://github.com/ariatemplates/ariatemplates could be put on the homepage.

**Comment by lsimone** [comment]
I agree, for sure we should change the landing page, because `JSObject methods` isn't the most useful page of the API docs;
I would definitely go for a home page with shortcuts and a small readme.

:+1:
